sbin/build.sh

```diff
@@ -580,6 +580,12 @@ configureZlibLocation() {
   fi
 }
 
+configureGtestLocation() {
+  if [ "${BUILD_CONFIG[OPENJDK_FEATURE_NUMBER]}" -ge 15 ]; then
+      addConfigureArg "--with-gtest=" "${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[WORKING_DIR]}/installedgtest"
```

Comment:
Maybe this should be optional? Yes, enabled by default, but allowed to be disabled?

Reply:
I'll think about it.